modules/unsupported/pom.xml

```diff
@@ -206,7 +218,8 @@
 				<module>solr</module>
 				<module>css</module>
 				<module>geojsonstore</module>
-                <module>s3-geotiff</module>
+        <module>s3-geotiff</module>
+        <module>gtopo30</module>

```

Comment:
I see spaces here, but tabs above, around line 59. While I would prefer spaces, tabs are currently more consistent with the current pom file.